source/iostreams.tex

```diff
@@ -13249,6 +13479,7 @@
 These functions enable use of \tcode{recursive_directory_iterator}
 with range-based for statements.
 
+\indexlibrary{\idxcode{begin}!\idxcode{recursive_directory_iterator}}%
```

Comment:
Why is there only one version of this index entry? Is this not a member function?